src/components/Map.tsx

```
@@ -1,11 +1,39 @@
+  iconSize: [24, 24],
+  iconAnchor: [12, 12],
+});
+
+const endIcon = divIcon({
+  html: '<div style="background-color: #ef4444; color: white; border-radius: 50%; width: 24px; height: 24px; display: flex; align-items: center; justify-content: center; font-weight: bold; font-size: 12px; border: 2px solid white; box-shadow: 0 2px 4px rgba(0,0,0,0.3);">E</div>',
+  className: "custom-marker",
+  iconSize: [24, 24],
+  iconAnchor: [12, 12],
+});
+
+const Map = ({ bounds, points, height = "400px", width = "400px", showMarkers = true }: MapProps) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)